pkg/kubelet/types/pod_update_test.go

```diff
@@ -114,6 +148,137 @@ func TestString(t *testing.T) {
 	}
 }
 
+func TestIsMirrorPod(t *testing.T) {
+	cases := []struct {
+		pod      *v1.Pod
+		expected bool
+	}{
+		{
+			pod:      getTestPod(configMirrorAnnotation(), nil),
+			expected: true,
+		},
+		{
+			pod:      getTestPod(nil, nil),
+			expected: false,
+		},
+	}
+	for i, data := range cases {
+		isMirrorPod := IsMirrorPod(data.pod)
+		assert.Equal(t, data.expected, isMirrorPod, "test[%d]", i)
+		t.Logf("Test case [%d]", i)
+	}
+}
+
+func TestIsStaticPod(t *testing.T) {
+	cases := []struct {
+		pod      *v1.Pod
+		expected bool
+	}{
+		{
+			pod:      getTestPod(configSourceAnnotation(FileSource), nil),
+			expected: true,
+		},
+		{
+			pod:      getTestPod(configSourceAnnotation(HTTPSource), nil),
+			expected: true,
+		},
+		{
+			pod:      getTestPod(configSourceAnnotation(ApiserverSource), nil),
+			expected: false,
+		},
+		{
+			pod:      getTestPod(nil, nil),
+			expected: false,
+		},
```

Comment:
Is there any difference between ApiserverSource and not? I do not think we need all of the testcases.